changelog/5255.breaking.1.rst

```diff
@@ -0,0 +1 @@
+:meth:`sunpy.map.CompositeMap.plot` now internally calls :meth:`sunpy.map.GenericMap.plot` and :meth:`sunpy.map.GenericMap.draw_contours`, which may affect the plot output of existing user code.
```

Comment:
Would be good to indicate how big these effects are (ie. are they minor changes at a pixel level that users don't need to worry about, or major changes)

Reply:
Impossible to say definitively.  For composite maps that are helioprojective and already co-oriented, the difference will be negligible.  For other composite maps, the changes could be drastic. (because the previous compositing was incorrect).